lib/WP_Auth0_Routes.php

```diff
@@ -204,7 +204,7 @@ protected function migration_ws_get_user() {
 			}
 
 			if ($user instanceof WP_Error) {
-				WP_Auth0_ErrorManager::insert_auth0_error( 'migration_ws_get_user', $user );
+				WP_Auth0_ErrorManager::insert_auth0_error( __METHOD__, $user );
```

Comment:
In other places https://github.com/auth0/wp-auth0/pull/384/files#diff-277e60f8ce82a93a9c30ed5aa4c01dd9R189

You will show the name of the method called but not in this one e.g. `get_user_by` , does it have a `get_error_message` also available?

Reply:
@cocojoe - Actually ... that function looks for a `WP_Error` object and handles nicely:

https://github.com/auth0/wp-auth0/blob/master/lib/WP_Auth0_ErrorManager.php#L7